presqt/api_v1/views/target.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from presqt.api_v1.serializers.target import TargetsSerializer, TargetSerializer
+
+
+class TargetsList(APIView):
+    """
+    **Supported HTTP Methods**
+
+    * Get: Retrieve summary representations of all Targets.
+    """
+    required_scopes = ['read']
+
+    def get(self, request):
+        """
+        Retrieve all Targets.
+
+
+        Returns
```

Comment:
If you are documenting with Gitbook, we should probably avoid the extra indentation since it doesn't truly comply with the NumPy doc standard.

Reply:
True True. In full Swagger mode.